packages/wdio-cli/src/interface.js

```diff
@@ -55,19 +48,58 @@ export default class WDIOCLInterface extends EventEmitter {
              */
             worker: {}
         }
-        this.clearConsole()
+
+        this.onStart()
+    }
+
+    onStart() {
+        this.log(chalk.bold(`\nExecution of ${chalk.blue(this.totalWorkerCnt)} spec files started at`), this.start.toISOString())
+        if (this.inDebugMode) {
+            this.log(chalk.bgYellow.black('DEBUG mode enabled!'))
+        }
+        if (this.isWatchMode) {
+            this.log(chalk.bgYellow.black('WATCH mode enabled!'))
+        }
+        this.log('')
     }
 
-    clearConsole () {
-        this.display = []
+    onSpecRunning(cid) {
+        this.onJobComplete(cid, this.jobs.get(cid), 0, chalk.bold.cyan('RUNNING'))
+    }
+
+    onSpecRetry(cid, job, retries) {
+        this.onJobComplete(cid, job, retries, chalk.bold.yellow('RETRYING'))
+    }
+
+    onSpecPass(cid, job, retries) {
+        this.onJobComplete(cid, job, retries, chalk.bold.green('PASSED'))
+    }
+
+    onSpecFailure(cid, job, retries) {
+        this.onJobComplete(cid, job, retries, chalk.bold.red('FAILED'))
+    }
+
+    onJobComplete(cid, job, retries, message) {
+        const details = []
+        if (job) {
+            const filename = Array.isArray(job.specs) ? job.specs.join(', ').replace(process.cwd(), '') : ''
+            const retryCount = retries > 0 ? `(${retries} retries)` : ''
+            details.push('in', getRunnerName(job.caps), filename ? `- ${filename}` : '', retryCount)
```

Comment:
It might be cleaner to put as argument just `job` variable and change `getRunnerName` to:
```js
export function getRunnerName ({caps = {}}) {
   ...
}
```

Reply:
Can't get what do you mean here. I can't modify getRunnerName function that way. It accepts capabilities object, not an object containing capabilities.
```{
 browserName: 'Chrome'
}```
not
```{
  capabilities: {
     browserName: 'Chrome'
  }
}
```

Reply:
but you insert argument like `job.caps`. So, if you will insert `job` object with `caps` property it will work.

Reply:
I think we are talking about different things here, getRunnerName doesn't accept job, it accepts capabilities